doc/release-notes.md

```diff
@@ -51,6 +51,14 @@ The following outputs are affected by this change:
 
 ### P2P protocol and network code
 
+Fee filtering of invs (BIP 133)
```

Comment:
Nit: I am pretty sure this goes up to "Notable changes" (`Example item`) as this place down here is usually used for the detailed change log only.